### DISCUSSION TITLE
support gRPC-Web proxy in Apache APISIX

### DISCUSSION THREAD

**Discussion by shuaijinchao** [grpc/grpc-web#1185] [Ideas]
Hi everyone, friends in the gRPC-Web community.

I come from the [Apache APISIX](https://github.com/apache/apisix) community. APISIX is a high-performance `API gateway`, specifically used to manage and schedule the flow of `NORTH-SOUTH` and `EAST-WEST`, or as a k8s ingress controller. We plan to support `gRPC-Web proxy` in APISIX. gRPC-Web solves the problem of using gRPC in the browser, allowing users to use gRPC services in a more native way. I think this is a cool thing for many front-end developers. If you are interested, welcome to discuss with us.

Proposal link:
https://lists.apache.org/thread/k6jtmdz147l0b236d77nhx197s3gxlcz

**Comment by sampajano** [comment]
Thanks for letting us know! Glad to hear you're adopting grpc-web :)

I'll convert this into a discussion thread so it'll be easier for anyone else interested to participate in :)

**Comment by yzeng25** [comment]
Hi @sampajano ,

Apache APISIX supports gRPC-Web now :) 
The blog is published on Apache APISIX's blog[1]. Can we restart this discussion soon?
Thanks!
[1]https://apisix.apache.org/blog/2022/01/25/apisix-grpc-web-integration

**Reply by sampajano** [reply]
@shuaijinchao @yzeng25 Thanks so much for the interest and updates! Glad to know you're integrating with grpc-web! :)

I'm curious if there are any specific topics that you're interested in discussing? Thanks :)

**Comment by sampajano** [comment]
https://github.com/grpc/grpc.io/issues/932 asked about potentially doing a blog post about APISIX integration.

I think for now what we could do is updating grpc-web documentation on mention APISIX proxy support. :)

I'll make the doc update soon. Thanks! :)

**Reply by shuaijinchao** [reply]
thx~ @sampajano

**Reply by yzeng25** [reply]
Awesome!
